backend/src/model/dto.rs

```diff
@@ -254,6 +266,18 @@ pub struct MapSearchParameters {
     pub is_inactive: Option<bool>,
     /// The owner of the map.
     pub owner_id: Option<i32>,
+    /// Wheter or not the map is private.
+    pub is_private: Option<bool>,
+}
+
+/// Support struct for transmitting latitude/longitude coordinates.
+#[typeshare]
+#[derive(Debug, Serialize, Deserialize)]
+pub struct LatLng {
+    /// Latitude of the point.
+    pub lat: f64,
+    /// Longitude of the point.
+    pub lng: f64,
```

Comment:
I would personally prefer then to be written out (latitude/longitude). If you want the short form then rename lng to long (that is the correct abbreviation according to wikipedia)